sms_sponsorship/controllers/main.py

```diff
@@ -124,6 +124,16 @@ def sms_change_child(self, child_request_id):
 
         sms_child_request.change_child()
 
+    @route('/sms_registration_confirmation/<model('
```

Comment:
Route should be ''/sms_sponsorship/step2/<model("recurring.contract"):sponsorship>/confirm" for consistency